UI.tsv

```diff
@@ -420,17 +420,17 @@ UI_20150317_000419	Number of Fragments
 UI_20150317_000420	{memo X}Parts 1
 UI_20150317_000421	{memo X}Parts 2
 UI_20150317_000422	{memo X}Parts 3
-UI_20150317_000423	Congratulations! Your level has increased.
+UI_20150317_000423	Congratulations! You have leveled up.
 UI_20150317_000424	Item Alarm
 UI_20150317_000425	{@st64}Details{/}
 UI_20150317_000426	How to Play
 UI_20150317_000427	${@st42b}          Move once with each of the keys{/}
-UI_20150317_000428	${@st42b}         Use a basic attack{/}
-UI_20150317_000429	${@st42b}                Jump up{/}
-UI_20150317_000430	{@st42b}     Use two keys together to move diagonally{/}
+UI_20150317_000428	${@st42b}         Perform a basic attack{/}
+UI_20150317_000429	${@st42b} Jump at the current location or while moving.{/}
```

Comment:
The extra spaces are there to make the text centered. You would have to make this change within the game using the realtime translation tool to see if your version is neatly centered. Is it?